src/seismometer/controls/explore.py

```diff
@@ -0,0 +1,1109 @@
+        """Generates the 6 plots for model evaluation"""
+        from seismometer._api import plot_model_evaluation
+
+        return plot_model_evaluation(
+            self.option_widget.cohorts,
+            self.option_widget.target,
+            self.option_widget.score,
+            list(self.option_widget.thresholds.values()),
+            per_context=self.option_widget.group_scores,
+        )
+
+    def generate_plot_code(self) -> str:
+        """Generates the code for the model evaluation plot"""
+        args = ", ".join(
+            [
+                repr(x)
+                for x in [
+                    self.option_widget.cohorts,
+                    self.option_widget.target,
+                    self.option_widget.score,
+                    list(self.option_widget.thresholds.values()),
+                ]
+            ]
+        )
+        return f"sm.plot_model_evaluation({args}, per_context={self.option_widget.group_scores})"
+
+
+@export
+class ExploreCohortEvaluation(ExplorationCohortSubclassEvaluationWidget):
+    """
+    A widget for exploring the model performance of a cohort.
+    """
+
+    def __init__(self):
+        """
+        Exploration widget for cohort evaluation.
+
+
+        Creates a 2x3 grid of individual performance metrics across cohorts.
+
+        Plots include Sensitivity, Flagged, PPV, Specificity, NPV vs Thresholds.
+        Includes a legend with cohort size.
+        """
+        from seismometer.seismogram import Seismogram
+
+        sg = Seismogram()
+        thresholds = {f"Threshold {k}": v for k, v in enumerate(sorted(sg.thresholds, reverse=True), 1)}
+        super().__init__(
+            "Cohort Group Performance",
+            sg.available_cohort_groups,
+            sg.target_cols,
+            sg.output_list,
+            thresholds=thresholds,
+            per_context=False,
+        )
+
+    def generate_plot(self) -> HTML:
+        """Generates the cohort evaluation plot"""
+        from seismometer._api import plot_cohort_evaluation
+
+        return plot_cohort_evaluation(
+            self.option_widget.cohort,
+            self.option_widget.cohort_groups,
+            self.option_widget.target,
+            self.option_widget.score,
+            list(self.option_widget.thresholds.values()),
+            per_context=self.option_widget.group_scores,
+        )
+
+    def generate_plot_code(self) -> str:
+        """Generates the code for the cohort evaluation plot"""
+        args = ", ".join(
+            [
+                repr(x)
+                for x in [
+                    self.option_widget.cohort,
+                    self.option_widget.cohort_groups,
+                    self.option_widget.target,
+                    self.option_widget.score,
+                    list(self.option_widget.thresholds.values()),
+                ]
+            ]
+        )
+        return f"sm.plot_cohort_evaluation({args}, per_context={self.option_widget.group_scores})"
```

Comment:
It was a bit of effort, but well worth it, now the Exploration* classes implement a `generate_plot_args` method, so that we can just pass in the actual plot function. I've added an example of this to the `classifier_bin` notebook. 

This also allows me to move all the custom Expore* objects directly into the `_api` module so cleanup startup some.